examples/platform/silabs/display/lcd.h

```diff
@@ -40,6 +40,7 @@ class SilabsLCD
         QRCodeScreen,
 #endif
         InvalidScreen,
+        CycleScreen,
```

Comment:
InvalidScreen should be the last entry of the enum

Reply:
Done